docs/en/ingest-management/elastic-agent/configuration/elastic-agent-monitoring.asciidoc

```diff
@@ -38,10 +38,9 @@ To enable monitoring, set `agent.monitoring.enabled` to `true`. Also set the
 collected. If neither setting is specified, monitoring is turned off. Set
 `use_output` to specify the output to which monitoring events are sent.
 
-Setting `agent.monitoring.enabled` to `true` also exposes a `/liveness` endpoint. By default, the endpoint 
-returns a `200` OK status as long as {agent}'s internal main loop is responsive and can 
-process configuration changes. It can be configured to also monitor the component states 
-and return an error if anything is degraded or has failed.
+You can also add the setting `agent.monitoring.enabled.http: true` to expose a `/liveness` endpoint.
```

Comment:
```suggestion
You can also add the setting `agent.monitoring.http.enabled: true` to expose a `/liveness` endpoint.
```
https://github.com/elastic/elastic-agent/blob/3500a81f766393c955a3d899a94bca813a4638de/elastic-agent.yml#L62-L90

You cannot set this from Fleet today in the UI, in 8.15.0 it will work with the overrides API at least, but today it can't be configured at all.